libs/core/src/lib/notification/notification-header/notification-header.component.html

```diff
@@ -2,6 +2,7 @@
 <ng-content select="[fd-notification-title]"></ng-content>
 <button
     fd-button
+    aria-label="close-notification"
```

Comment:
Should we use kebab-case here? This is normal text value that should be read by device (at least that's how I understand it).

Reply:
Shouldn't we have a way to add it in different language?

Reply:
fixed, the user will now decide like they do in shellbar :)

Reply:
Thanks Stefano, we can merge it :D